api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -175,6 +175,9 @@
   private static final String FITBIT_SQL =
```

Comment:
Should this be HAS_FITIBIT_SQL as well for consistency?

Reply:
Done.